src/main/kotlin/com/dansplugins/factionsystem/command/faction/claim/MfFactionClaimFillCommand.kt

```diff
@@ -89,6 +89,11 @@ class MfFactionClaimFillCommand(private val plugin: MedievalFactions) : CommandE
                             sender.sendMessage("$RED${plugin.language["CommandFactionClaimFillNoClaimableChunks"]}")
                             return@saveChunks
                         }
+                        val claimFillMaxChunks = plugin.config.getInt("factions.claimFillMaxChunks", -1)
```

Comment:
Bukkit API shouldn't be accessed asynchronously - this needs to be pulled out of the async task

Reply:
This call has been pulled out of the async task.

I feel like I've asked you this before but why shouldn't the Bukkit API be accessed asynchronously?